scripts/install.js

```diff
@@ -150,6 +150,43 @@ function getTempFile(cached) {
     .slice(2)}.tmp`;
 }
 
+function validateChecksum(tempPath, name) {
+  let storedHash;
+  try {
+    const checksums = fs.readFileSync(path.join(__dirname, '../checksums.txt'), 'utf8');
+    const entries = checksums.split('\n');
```

Comment:
Should we split up `fs` operation and the checksum string operations into two separate try catches? I think if the read file operation fails we should throw an error.

Reply:
It will fail when used locally on purpose, as there are no checksums in the repo, so we cannot throw then